### DISCUSSION TITLE
Is Moko Resources still under active development?

### DISCUSSION THREAD

**Discussion by kieren-harvest** [icerockdev/moko-resources#434] [General]
Moko Resources is a great resource, but it's been several months since any changes have been made to the master or develop branches.

Before integrating it into a project it would be great to know if the project is still supported.

Specifically I'm interested in the fix for issue #355 which should appear in the next release from what I can tell.

**Comment by Alex009** [comment]
hi.
we continue maintaining of moko libs, but not have active development. also we plan to set aside a day a week for moko support
